code/modules/ninja/suit/mask.dm

```diff
@@ -10,130 +10,9 @@ Contents:
 
 
 
-/obj/item/clothing/mask/gas/voice/space_ninja
+/obj/item/clothing/mask/gas/space_ninja
 	name = "ninja mask"
 	desc = "A close-fitting mask that acts both as an air filter and a post-modern fashion statement."
 	icon_state = "s-ninja"
 	item_state = "s-ninja_mask"
-	vchange = 1
 	strip_delay = 120
-
-/obj/item/clothing/mask/gas/voice/space_ninja/speechModification(message)
```

Comment:
>The mask itself works fine. Leave it be.

it isn't compatible with the changes and I can't be assed to snowflake out the chameleon functionality for an already snowflakey voice changer special snowflake subtype for NINJA.